canvas_modules/common-canvas/src/common-properties/components/virtualized-table/virtualized-table.scss

```diff
@@ -184,6 +176,8 @@
 	}
 
 	.properties-vt-row-selected {
```

Comment:
@matthoward366 Added style here to retain the style until the user click somewhere else. `:focus` & `:active` , is not retaining the style after clicking, and shows at the clicking moment only. 
Please let me know what looks best.